### DISCUSSION TITLE
什么是3A游戏

### DISCUSSION THREAD

**Discussion by dfddtabc** [share121/inter-knot#2986] [其他]
![屏幕截图 2024-08-19 095530](https://github.com/user-attachments/assets/acf27565-fdc3-4eb3-8b06-b93c9e3165f6)

30级前：指三个人轮流a
40级后：被怪轮流a三个人。

**Comment by dfddtabc** [comment]
该图片来自b站up主：枫丹水母头璃月梅花眼
视频链接：https://www.bilibili.com/video/BV1axePe3EAY/?spm_id_from=333.337.search-card.all.click

**Reply by dfddtabc** [reply]
好像不能发连接

**Reply by LT-UT** [reply]
可以

**Comment by chihung3927** [comment]
喜欢我的双闪吗!
